libs/common/utilities/src/css/icon/icon.css.ts

```diff
@@ -0,0 +1,60 @@
+import { css, CSSResult, unsafeCSS } from 'lit';
+import iconMapping from './icon-mapping.css';
+import { focusOutline } from '../mixin/outline.mixin.css';
+
+const styles: CSSResult = css`
+    ${unsafeCSS(iconMapping)}
+
+    :host {
+        display: inline-flex;
+    }
+
+    /* Icon styles - gebaseerd op DV _icon.scss */
+    .vl-icon {
+        font-family: var(--vl-icon-font);
+        font-size: 1em;
+        line-height: 1;
+        display: inline;
+        color: inherit;
+
+        &.vl-icon--small {
+            font-size: 0.8em;
+        }
+
+        &.vl-icon--large {
+            font-size: 1.2em;
+        }
+
+        &.vl-icon--light {
+            color: var(--vl-light-text-color);
+        }
```

Comment:
Graag `vl-icon--light` styling onder `vl-icon--clickable` plaatsen, dan is het mogelijk `light`-styling te combineren met `clickable` styling. Nu is er een css specificity conflict. (Eventueel te bespreken als je die combinatie liever niet als mogelijkheid houdt voor de afnemer.)

![clickable-light](https://github.com/milieuinfo/uigov-web-components/assets/15731348/f4c30995-c1e7-4438-add4-ae55fa433420)

Reply:
Die combinatie moet niet mogelijk zijn. Een clickable icon heeft een specifieke kleur die we afdwingen.